runtime/lua/vim/lsp/_inlay_hint.lua

```diff
@@ -17,11 +17,10 @@ local hint_cache_by_buf = setmetatable({}, {
 
 local namespace = api.nvim_create_namespace('vim_lsp_inlayhint')
 
-M.__explicit_buffers = {}
+M.__buffers = {}
```

Comment:
is this used outside this module any more?

Reply:
done